### DISCUSSION TITLE
Правила валидации Validation rules

### DISCUSSION THREAD

**Discussion by Kenny201** [livewire/livewire#4202] [1. Help] [answered]
Where can I find out or read about the lists of validation rules that I can specify in the rules array?
What do I mean by that? For example, there are rules nullables, image, max, etc. So the question is, is there a list of all these rules that I can use?

 protected $rules = [
        'title' => 'required',
        'content' => 'required',
        'photo' => 'nullable|sometimes|image|max:5000',
    ];

**Comment by ralphjsmit** [comment, marked as answer]
Hey there, the validation rules are provided by Laravel and not by Livewire. The Laravel docs have an excellent section with all the [available validation rules](https://laravel.com/docs/8.x/validation#available-validation-rules) and a good explanation of every rule.

Hope that helps!

**Reply by Kenny201** [reply]
Here I am a fool, thank you very much, very helpful)

**Reply by ralphjsmit** [reply]
Happy to hear that! Could you mark this thread as answered, so that it's not shown as unanswered anymore on the forum page?

**Reply by Kenny201** [reply]
For some reason there is no mark as an answer button or maybe I'm not looking there

**Reply by Kenny201** [reply]
Since this discussion is not a Q&A, there is no answer button here)

**Reply by ralphjsmit** [reply]
Ah lol, that's true!💪

**Comment by joshhanley** [comment]
Category was set to "show and tell" I have changed this to "help" and marked it as answered 🙂
